src/System.IO.IsolatedStorage/tests/System.IO.IsolatedStorage.Tests.builds

```diff
@@ -3,15 +3,9 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.IO.IsolatedStorage.Tests.csproj">
-      <OSGroup>Windows_NT</OSGroup>
```

Comment:
I'd be interested in @ericstj thoughts on this but yes I would prefer we keep it for consistency right now and we can decide to fold them all later if needed.

Also with the current settings in this .builds file (i.e. no OSGroup here) you will have a bin-clash between the 2 configurations, as the build options (OSGroup/TargetGroup) are the same for 2 of these configurations. The TestTFMs is not part of the build configuration so don't get factored into the output directories of the build, thus we cannot have configurations that only differ by TestTFMs.